c10/util/Float8_e4m3fnuz-inl.h

```diff
@@ -12,21 +13,208 @@ namespace c10 {
 
 /// Constructors
 
-C10_HOST_DEVICE inline Float8_e4m3fnuz::Float8_e4m3fnuz(float value)
+inline C10_HOST_DEVICE Float8_e4m3fnuz::Float8_e4m3fnuz(float value)
     : x(detail::fp8e4m3fnuz_from_fp32_value(value)) {}
 
 /// Implicit conversions
 
-C10_HOST_DEVICE inline Float8_e4m3fnuz::operator float() const {
+inline C10_HOST_DEVICE Float8_e4m3fnuz::operator float() const {
   return detail::fp8e4m3fnuz_to_fp32_value(x);
 }
 
 /// Special values helper
 
-C10_HOST_DEVICE inline bool Float8_e4m3fnuz::isnan() const {
+inline C10_HOST_DEVICE bool Float8_e4m3fnuz::isnan() const {
   return x == 0b10000000;
 }
 
+/// Arithmetic
```

Comment:
note: can we remove this? context: https://github.com/pytorch/pytorch/issues/113663, this should not have been added to the other float8 flavors.

Reply:
basically this entire section can be removed, unless this is needed for real use cases

Reply:
Should we remove the arithmetic from the other F8 flavors as part of this PR, or a follow-up PR?

Reply:
I'd vote separate PR to keep things small